src/components/Link/Link.jsx

```diff
@@ -3,32 +3,39 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import PropTypes from 'prop-types'
 import './link.css'
 
-const Link = ({ addClass, href, icon, label, style, ...props }) => (
-  <a
-    href={href}
-    className={`link ${addClass}`}
-    style={style}
-    {...props}
-  >
-    {icon && (
-      <FontAwesomeIcon icon={icon} className={`small ${label.length > 0 && 'me-2'}`} />
-    )}
-    {label}
-  </a>
-)
+const Link = ({ addClass, href, icon, label, style, ...props }) => {
+  return (
+    <a
+      href={href}
+      className={`link ${addClass}`}
+      style={style}
+      {...props}
+    >
+      {icon && (
+        <FontAwesomeIcon icon={icon} className={`small ${label.length && 'me-2'}`} />
+      )}
+      {label && label}
+    </a>
+  )
+}
 
+/**
+ * there isn't an easy way to require either the icon OR the label. there are additional packages we could install that have the
+ * capability, but that would bloat the package unnecessarily. instead, we're requiring that the label be passed. the overwhelming
+ * majority of instances where a link is used, will want a label anyway. however, in the event a label isn't desired, passing the
+ * value as an empty string will satisfy prop types, while still only showing an icon.
```

Comment:
I tried [this suggestion](https://github.com/scientist-softserv/webstore-component-library/issues/165#issuecomment-1438914609), but it felt like overkill.